ext/oci8/oci8.stub.php

```diff
@@ -2,534 +2,468 @@
+function oci_lob_tell(OCILob $lob): int|false {}
 
-function oci_lob_rewind(OCILob $lob_descriptor): bool {}
+function oci_lob_rewind(OCILob $lob): bool {}
 
-function oci_lob_seek(OCILob $lob_descriptor, int $offset, int $whence = OCI_SEEK_SET): bool {}
+function oci_lob_seek(OCILob $lob, int $offset, int $whence = OCI_SEEK_SET): bool {}
 
-function oci_lob_size(OCILob $lob_descriptor): int|false {}
+function oci_lob_size(OCILob $lob): int|false {}
 
-function oci_lob_write(OCILob $lob_descriptor, string $string, ?int $length = null): int|false {}
+function oci_lob_write(OCILob $lob, string $data, ?int $max_length = null): int|false {}
 
-function oci_lob_append(OCILob $lob_descriptor_to, OCILob $lob_descriptor_from): bool {}
+function oci_lob_append(OCILob $to, OCILob $from): bool {}
 
-function oci_lob_truncate(OCILob $lob_descriptor, int $length = 0): bool {}
+function oci_lob_truncate(OCILob $lob, int $length = 0): bool {}
 
-function oci_lob_erase(OCILob $lob_descriptor, ?int $offset = null, ?int $length = null): int|false {}
+function oci_lob_erase(OCILob $lob, ?int $offset = null, ?int $length = null): int|false {}
 
-function oci_lob_flush(OCILob $lob_descriptor, int $flag = 0): bool {}
+function oci_lob_flush(OCILob $lob, int $flag = 0): bool {}
 
-function ocisetbufferinglob(OCILob $lob_descriptor, bool $mode): bool {}
+function ocisetbufferinglob(OCILob $lob, bool $mode): bool {}
 
-function ocigetbufferinglob(OCILob $lob_descriptor): bool {}
+function ocigetbufferinglob(OCILob $lob): bool {}
 
-function oci_lob_copy(OCILob $lob_descriptor_to, OCILob $lob_descriptor_from, ?int $length = null): bool {}
+function oci_lob_copy(OCILob $to, OCILob $from, ?int $length = null): bool {}
 
-function oci_lob_is_equal(OCILob $lob_descriptor_first, OCILob $lob_descriptor_second): bool {}
+function oci_lob_is_equal(OCILob $lob1, OCILob $lob2): bool {}
 
-function oci_lob_export(OCILob $lob_descriptor, string $filename, ?int $start = null, ?int $length = null): bool {}
+function oci_lob_export(OCILob $lob, string $filename, ?int $offset = null, ?int $length = null): bool {}
 
 /**
  * @alias oci_lob_export
  * @deprecated
  */
-function ociwritelobtofile(OCILob $lob_descriptor, string $filename, ?int $start = null, ?int $length = null): bool {}
+function ociwritelobtofile(OCILob $lob, string $filename, ?int $offset = null, ?int $length = null): bool {}
 
-/**
- * @param resource $connection_resource
- */
-function oci_new_descriptor($connection_resource, int $type = OCI_DTYPE_LOB): ?OCILob {}
+/** @param resource $connection */
+function oci_new_descriptor($connection, int $type = OCI_DTYPE_LOB): ?OCILob {}
 
 /**
- * @param resource $connection_resource
+ * @param resource $connection
  * @alias oci_new_descriptor
  * @deprecated
  */
-function ocinewdescriptor($connection_resource, int $type = OCI_DTYPE_LOB): ?OCILob {}
+function ocinewdescriptor($connection, int $type = OCI_DTYPE_LOB): ?OCILob {}
```

Comment:
Oh wait! we did use the provider's name as the connection parameter in case of a few other extensions (`$mysql`/`$sqlite3` etc.), and I forgot about this here. So should I rather use `$oci`/`$oci8`, right? I guess if we converted the connection resource to a class, we'd choose `OciConnection`, so probably the former makes more sense.

Reply:
`$connection` seems more natural, since the verbal description of `oci_connect()` is that it 'returns a connection to the database'.

Reply:
I'm happy to keep this at `$connection` here.

Reply:
Ok, then it's decided. I'm also perfectly fine with `$connection`.